src/components/sign.rs

```diff
@@ -13,6 +13,7 @@ use crate::{
     util::{
         days_to_blocks, hours_to_blocks, parse_escrow_type, parse_network, parse_npub, parse_nsec,
     },
+    validation::*,
```

Comment:
no globbing import the stuff explicitly, this is not Python...
We only glob preludes like `dioxus::prelude::*` or in tests with `use super::*`